### ISSUE TITLE
Procedure partial queries

### ISSUE THREAD

**Issue by dgasmith** [MolSSI/QCFractal#128]
We will need the ability to perform partial queries on procedures. An example of this is as follows:

1) We have a *n* TorsionDrives which "own" *m* optimization runs.
2) Pulling all *m* optimizations for all *n* TorsionDrive can be too much data.
3) The TorsionDrive only wants to know about several quantities these are:
 - The `final_molecule` of each optimization
 - The `final_energy` of each optimization
 - The `final_result` of each optimization

Planned implementation:

```
GET: procedures/optimization/

Body:
{
    "meta":{},
    "data": {"ids": List[str],
             "quantity": ["final_molecule", "final_energy", "final_result", ...]}
}

Return:
{
    "meta":{"normal_data"}
    "data": Dict["id", quantity]
}
```

**Comment by mattwelborn** [comment]
Is this closed by #393?

**Comment by dgasmith** [comment]
Yup.
